packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -211,7 +234,11 @@ export function ArticleMetadataPanel({
                     value: author.id,
                     label: author.name
                   }))}
-                  onSearch={searchKeyword => setAuthorsFilter(searchKeyword)}
+                  onSearch={searchKeyword => {
+                    setAuthorsFilter(searchKeyword)
+                    setAuthorName(searchKeyword)
```

Comment:
Do you even need a state for `authorName` you could also take the value from `authorsFilter`.